src/rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+  end
```

Comment:
- [ ] Kindly make implementation for the rental that is adding itself to the @book and @person ** in order to achieve the 'belongs-to' association**

<img width="497" alt="g126" src="https://github.com/alexansaa/School-Library/assets/103386183/5b50fb84-95dc-48cd-91e3-2681a8c620f4">